### PR TITLE
Feature specs

### PR DESCRIPTION
**Pull request by Morantron**
Resurrecting https://github.com/coopdevs/timeoverflow/pull/254

### CODE REVIEW

**Comment by sauloperez** [comment]
❤

File: spec/features/sign_in_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  context 'with a valid password' do
+    it 'signs the user in' do
+      expect(Capybara.current_session.driver.browser.manage.cookie_named('_timeoverflow_session')).to be_falsy
+
+      sign_in_with(user.email, 'papapa22')
+
+      expect(Capybara.current_session.driver.browser.manage.cookie_named('_timeoverflow_session')).to be_truthy
+    end
```
Comment (sauloperez) [inline spec/features/sign_in_spec.rb]: I would also expect to navigate to a page which contains the nav menu, or the terms of service

File: spec/features/transfer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    expect(page).to have_css('.transactions tr', count: 1, text: '2:00')
+  end
+
+  def submit_transfer_form_with(hours: nil, minutes: nil)
+    within transfer_form do
+      fill_in 'transfer_hours', with: hours
+      fill_in 'transfer_minutes', with: minutes
+
+      # hack alert! there is no translation for this string. How we build the
+      # copy then?
```
Comment (sauloperez) [inline spec/features/transfer_spec.rb]: I'd rather create an issue

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -54,28 +67,58 @@
...
+  config.include Devise::TestHelpers, type: :controller
+  config.include ControllerMacros, type: :controller
+  config.include Features::SessionHelpers, type: :feature
+
+  # Create terms and conditions
+  config.before do
+    Document.create!(label: "t&c") do |doc|
+      doc.title = "Terms and Conditions"
+      doc.content = "blah blah blah"
+    end
```
Comment (sauloperez) [inline spec/spec_helper.rb]: What about creating a `#create_terms_and_conditions`?

File: spec/features/transfer_spec.rb (hunk 2 above)
Comment (Morantron) [reply spec/features/transfer_spec.rb]: this is just active record magic, [see here](https://github.com/coopdevs/timeoverflow/blob/6d069bcf46a234fe8a65ed7955f6f02ffb0c4846/app/views/transfers/new.html.erb#L39)

File: spec/features/create_offer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  context 'create an offer' do
+    it 'can be created' do
+      sign_in_with(user.email, user.password)
+      click_on I18n.t('activerecord.models.offer.other')
+      click_on I18n.t('helpers.submit.create', model: I18n.t('activerecord.models.offer.one'))
+      fill_in 'offer_title', with: 'Le title'
+      fill_in 'offer_description', with: 'Lorem ipsum in the night'
+      select category.name, from: 'offer_category_id'
+
+      # TODO there are two i18n keys for getting "Crear oferta" copy ( one returns 'Crear oferta' and the other 'Crear Oferta' )
```
[the remark this answers is not in the document]
Comment (enricostano) [reply spec/features/create_offer_spec.rb]: Done https://github.com/coopdevs/timeoverflow/issues/352

**Pull request merged**